cmd/lookoutd/serve.go

```diff
@@ -140,6 +143,39 @@ func (c *ServeCommand) Execute(args []string) error {
 	return server.NewServer(watcher, poster, dataHandler.FileGetter, analyzers, eventOp, commentsOp).Run(ctx)
 }
 
+func (c *ServeCommand) logConfig(conf Config) {
+	cCp := ServeCommand{
+		ConfigFile:    c.ConfigFile,
```

Comment:
I'm pretty sure sooner or later somebody (read: me) will forget to update the struct here when changing ServeCommand.

How about just to deep-copy it and rewrite necessary fields?
You can use this lib for it for example: https://github.com/jinzhu/copier

Reply:
I think we have to make a compromise here. We either risk forgetting to add new fields, or we risk forgetting to remove new fields with sensitive info.

I thought I'd go with the safest way. But I could change it for the deep copy if you guys think that would be better. We probably won't be adding many fields with credentials or secrets.

Reply:
I can see only 2 such new field: private key and queue connection string. But we will definetely have more configuration options. The thing is: it's easy to spot sensetive fields in logs and fix it. Not that easy to spot missing fields.

Reply:
That's the problem, sensitive fields should not be easy to spot in the logs :D
Anyway, I made the change in cbe937d, let's keep an eye on new fields to make sure we don't leak anything.

Reply:
haha. I mean it's easy to spot during development. You would see it when you run the server locally and kill it.